packages/component-library/stories/CivicCard.story.js

```diff
@@ -0,0 +1,335 @@
+          vulnerable populations but have not yet experienced demographic
+          changes. Their housing market is still low or moderate but has
+          experienced high appreciation since 2008 (or 2012 for rents).
+        </p>
+        <p>
+          <em>Early: Type 2: </em>These neighborhoods have higher shares of
+          vulnerable populations but have experienced demographic changes
+          whereby they are losing vulnerable populations proportionally. Their
+          housing market is low or moderate, but they are adjacent to tracts
+          whose values are already high or are increasing rapidly.
+        </p>
+        <h4>MID-STAGE GENTRIFICATION</h4>
+        <p>
+          <em>Dynamic: </em>These neighborhoods are currently undergoing
+          gentrification. They have higher shares of vulnerable populations but
+          have experienced demographic changes by losing vulnerable populations
+          proportionally. Their housing market is still low or moderate but has
+          experienced high appreciation since 2008 (or 2012 for rents).
+        </p>
+        <h4>LATE-STAGE GENTRIFICATION</h4>
+        <p>
+          These neighborhoods have mostly gentrified but vulnerable populations
+          may still reside in there. The housing market has completely shifted
+          from low or moderate to high value.
+        </p>
+        <p>
+          <em>Late: Type 1: </em>These neighborhoods have higher shares of
+          vulnerable populations but have experienced demographic changes by
+          losing vulnerable populations proportionally. Their housing market
+          used to be low or moderate in 2000 but has appreciated rapidly since,
+          and now values are high.
+        </p>
+        <p>
+          <em>Late: Type 2: </em>A new typology in 2018, these neighborhoods no
+          longer have high shares of vulnerable populations like they used to in
+          2000 or in 2006-10. They have experienced demographic changes by
+          losing their once-high share of vulnerable populations. Their housing
+          market is still low or moderate but has experienced high appreciation
+          since 2008 (or 2012 for rents).
+        </p>
+        <p>
+          <em>Continued Loss: </em>These neighborhoods no longer have high
+          shares of vulnerable populations like they used to in 2000 or in
+          2006-10. The share of white people is growing and/or the share of
+          people with a four-year degree is growing. Their housing market used
+          to be low or moderate in 2000 but has appreciated rapidly since, and
+          now values are high.
+        </p>
+      </Collapsable.Section>
+    </Collapsable>
+  ),
+  metadata: (
+    <p>
+      <em>
+        Note: This dataset documentation is for demonstration purposes only and
+        is not peer-reviewed.
+      </em>
+    </p>
+  ),
+  // metadataQA likely to take metadata API endpoint in the near future
+  metadataQA: [
```

Comment:
When running package 2018, the `TemplateCard` is now erroring out because it doesn't have `metadataQA`, is this property mandatory. If so, we should update the metadata for that card. If not, we should make sure that optional properties aren't interpreted as required.

Reply:
That being said, we should add an example of this to the README